### ISSUE TITLE
Midpoint Review - Design Feedback - BAH-MBS-SelfServ, Medallions Self-Service, Medallions 1330m Digitization and Integration to CaMEO

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#99096]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] Must: The [intro page template](https://design.va.gov/templates/forms/introduction#authenticated) has been updated since you last worked on this. Align your UI with the template. Add an action link under the info alert for authenticated. For unauthenticated, change the prefill alert to match the [content in the design system.](https://design.va.gov/patterns/help-users-to/know-when-their-information-is-prefilled#unauthenticated-prefill-alert).
- [ ] Must: Our prefill information has also recently updated. It is still in [draft mode in the design system](https://dev-design.va.gov/3562/patterns/help-users-to/update-prefilled-information), however the UI is correct in the [Figma file](https://www.figma.com/design/1z3bAkQl4uR1IvAxmtyqZi/AE-Design-Patterns---Update-Prefill?node-id=4013-14358&node-type=frame&t=dzbalmipmUCXVoaX-0). Compare your prefill to the Figma file and try to align the UI. You no longer need the blue summary box on each page about prefill, research showed that having the alert at the beginning was good enough and the summary box throughout the app was overwhelming. You can show the alert when a user is edting.

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by carly-yamrus** [comment]
<img width="225" alt="Image" src="https://github.com/user-attachments/assets/22393e61-d0ac-4f4e-9042-bbae585df456" />

Hi @allison0034   - for the pre-fill pattern, this now is using the cards that are editable. I know this is new, is it required to update this pre-filled screen to the card version?

**Comment by allison0034** [comment]
It would be flagged at Staging. It would not be launch blocking right now,
but it would be something you would need to fix in the future.

**Comment by allison0034** [comment]
You would not need to ask for DOB if it is not on the form. So if the cemetery pages do not require DOB, you do not need to add them. The pattern we have is for the Veteran.

**Comment by allison0034** [comment]
I synced with Carly and for the DOB, the form also asks for date of death and city so it is ok to deviate since the form has different requirements.

**Comment by allison0034** [comment]
Yes! For future reference, all recordings will be in your collab cycle ticket and then under each section (eg Midpoint Review).

[Midpoint review meeting recording](https://oddball-io.zoom.us/rec/share/tISqw2uAxWuPDyB7z60OYfjGM96VRGWDUAP4MnoX_I_EcIsRJX-7SMxor1a7i-jy.0U13bwQ3xIfDbhdH)
Passcode: Df=m9XK=

**Comment by carly-yamrus** [comment]
@allison0034 I think that's the wrong recording - it says "Staging Review - MHV Medical Records, Phase 1 Download Records, Flows/images, Settings" !!

**Comment by carly-yamrus** [comment]
> Must: We have added a pattern around [gender identity](https://dev-design.va.gov/3562/patterns/ask-users-for/gender-identity), however, per our guidance, you should only ask a user for their race or ethnicity, sexual orientation, or gender identity if you can clearly explain how providing this information will benefit the user (not the VA) and the question is related to the larger action they are trying to perform. If you are collecting for statistics this doe not seem to fit the need and should be removed. I know there has been a lot of discussion around this so if you get an update just comment on this ticket.

For the race/ethnicity/sex question, we have confirmation from @humancompanion-usds that the demographics sections is okay with just the _race and ethnicity questions._ It will not include sex or gender questions.

**Comment by carly-yamrus** [comment]
@allison0034 Hi Allison - we are completed on this ticket
